bundle-size/api.js

```diff
@@ -171,6 +171,44 @@ function choosePotentialApproverTeams(
  * @param {!GitHubUtils} githubUtils GitHubUtils instance.
  */
 exports.installApiRouter = (app, db, githubUtils) => {
```

Comment:
Just to clarify, `api.js` uses a GitHub API client with ampprojectbot auth, and `GitHubUtils` uses another GitHub API client with this user-based auth? Why do we need 2 different auths?

Reply:
As we are all very well aware, the GitHub API documentation is lacking... There are certain actions that app-based tokens can't perform, no matter the permissions we give them - any action relating to Organization (such as translating team names to team numbers and members) can only be performed with a user-based token.
Now it turns out that creating a review request cannot be performed with the user-based token, so I'm switching it back to the app-based token.
I wish it wasn't so. Or at least better documented...